src/app/components/team/SuggestedMatches.js

```diff
@@ -27,17 +27,18 @@ const SuggestedMatches = ({ routeParams }) => (
       render={({ error, props }) => {
         if (!error && props) {
           const { team } = props;
+          // Should we discard savedQuery already?
```

Comment:
I think we can get rid of `savedQuery` already.

Reply:
Done